diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -413,9 +413,10 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
 
     private fun ASTNode.isSingleDotStatementOnSingleLine() = parents()
         .takeWhile { it.elementType in expressionTypes }
-        .singleOrNull()
+        .takeIf { it.toList().size == 1 }
+        ?.single()
```

Comment:
This is basically same as before. I thought we need to check number of lines for case when `size == 0` too? If not, you can just leave `singleOrNull` and `?: false` at the end.